src/main/scala/mesosphere/marathon/upgrade/AppStartActor.scala

```diff
@@ -22,30 +22,32 @@ class AppStartActor(
     val instanceTracker: InstanceTracker,
     val eventBus: EventStream,
     val readinessCheckExecutor: ReadinessCheckExecutor,
-    val app: AppDefinition,
+    val runSpec: RunnableSpec,
     val scaleTo: Int,
     promise: Promise[Unit]) extends Actor with ActorLogging with StartingBehavior {
 
   val nrToStart: Int = scaleTo
 
   def initializeStart(): Unit = {
-    scheduler.startApp(driver, app.copy(instances = scaleTo))
+    // TODO (pods): there's no copy on a trait. we could pass the scaleTo instead
+    scheduler.startApp(driver, runSpec.copy(instances = scaleTo))
```

Comment:
I handled this already by adding a "withInstances" to the trait and then implemented it with copy.